sklearn/preprocessing/_discretization.py

```diff
@@ -63,6 +65,21 @@ class KBinsDiscretizer(TransformerMixin, BaseEstimator):
 
         .. versionadded:: 0.24
 
+    subsample : int or None (default='warn')
+        Maximum number of samples, used to fit the model, for computational
+        efficiency. Used when `strategy="quantile"`.
+        subsample=None means that all the training samples are used when
+        computing the quantiles that determine the binning thresholds.
+        Since quantile computation relies on sorting each column of X and
+        that sorting has an n log(n) time complexity, it is recommended
+        to use subsampling on datasets with a very large number of samples.
```

Comment:
I get a commit error with the math notation:

flake8...................................................................Failed
- hook id: flake8
- exit code: 1

sklearn/preprocessing/_discretization.py:75:36: W605 invalid escape sequence '\m'
sklearn/preprocessing/_discretization.py:75:50: W605 invalid escape sequence '\l'

Reply:
we can maybe just keep the code notation
```
`O log(n)`
```

This is more readable when reading the raw docstring and the rendering would be ok in the HTML

Reply:
My bad, this syntax might solely work for reStructuredText files and might need characters to be escaped otherwise -- which apart after having been HMTL-rendered is not nice to read.

Reply:
we might need to declare the string as a raw string:

```
r"""
The docstring with some math inside.
"""